app/kubemci/cmd/create_test.go

```diff
@@ -127,10 +127,15 @@ func TestCreateIngressAndLoadBalancer(t *testing.T) {
 	if err := ingress.UnmarshallAndApplyDefaults("../../../testdata/ingress.yaml", "" /*namespace*/, &originalIngress); err != nil {
 		t.Fatalf("%s", err)
 	}
-	validator := validations.NewFakeValidator()
+	validator := validations.NewFakeValidator(true /*validationShouldPass*/)
 
-	// TODO(G-Harmon): Verify that validation errors are propagated back.
-	createIngressAndLoadBalancer(originalIngress, clients, &options, nil /*cloudInterface*/, validator)
+	err := createIngressAndLoadBalancer(originalIngress, clients, &options, nil /*cloudInterface*/, validator)
+
+	// Test for urrent error, because not all of the resources are faked out:
+	expectedErr := "will not overwrite Ingress resource in cluster 'cluster1' without --force"
```

Comment:
Sorry I dont see why should it return this error. client is not returning any existing ingress